src/SDKs/Network/Management.Network/Properties/AssemblyInfo.cs

```diff
@@ -7,7 +7,7 @@
 [assembly: AssemblyTitle("Microsoft Azure Network Management Library")]
 [assembly: AssemblyDescription("Provides Microsoft Azure Network management functions for managing the Microsoft Azure Network service.")]
 
-[assembly: AssemblyVersion("10.0.0.0")]
+[assembly: AssemblyVersion("10.1.0.0")]
```

Comment:
@irrogozh please make similar updates to .csproj. Change in version in .csproj will what bump up the nuget package version.
Also delete all instances of project.json, global.json and *.xproj